### ISSUE TITLE
Error running check_and_gen.py script:

### ISSUE THREAD

**Issue by slowPoke26** [mxrch/GHunt#296]
Greetings,

While trying to run this with “docker run -v ghunt-resources:/usr/src/app/resources -ti ghcr.io/mxrch/ghunt check_and_gen.py”, I am having the following error:

**[Webdrivers Manager] I can't find the chromedriver, so I'm downloading and installing it for you...
Traceback (most recent call last):
  File "check_and_gen.py", line 220, in <module>
    driverpath = get_driverpath()
  File "/usr/src/app/lib/utils.py", line 228, in get_driverpath
    driver_path = ChromeDriverManager(path="/usr/src/app").install()
  File "/usr/local/lib/python3.8/site-packages/webdriver_manager/chrome.py", line 32, in install
    driver_path = self._get_driver_path(self.driver)
  File "/usr/local/lib/python3.8/site-packages/webdriver_manager/manager.py", line 31, in _get_driver_path
    binary_path = self.driver_cache.save_file_to_cache(file, browser_version,
  File "/usr/local/lib/python3.8/site-packages/webdriver_manager/driver_cache.py", line 27, in save_file_to_cache
    archive = save_file(file, path)
  File "/usr/local/lib/python3.8/site-packages/webdriver_manager/utils.py", line 40, in save_file
    with open(archive_path, "wb") as code:
PermissionError: [Errno 13] Permission denied: '/usr/src/app/drivers/chromedriver/linux64/96.0.4664.45/driver.zip’**

What am I missing? I used do run this a few months ago, but decided to try the latest build, so... Now I am stuck at this step.

Regards,

**Comment by mxrch** [comment]
Hey @slowPoke26 ! I can confirm the issue, can you check if the https://github.com/mxrch/GHunt/commit/d72177ecb278afe08138262bba1cbcb28f97ebb2 commit has fixed the issue ?
It fixed it for me, you can delete the image & container locally and download the latest version (I hardcode the sha1 digest here so you're sure to have the latest image) :
```bash
docker run -v ghunt-resources:/usr/src/app/resources -ti ghcr.io/mxrch/ghunt@sha256:77c8caa4ced5ac74c73e437b06d0dd963825e90af132c9d45ec136c842b1d31b check_and_gen.py
```

**Comment by slowPoke26** [comment]
Hey @mxrch !

Thanks for the fast response. The execution worked this time. It went trough the following steps:

**[Webdrivers Manager] I can't find the chromedriver, so I'm downloading and installing it for you...
[Webdrivers Manager] The chromedriver has been installed !**

I was able to set the cookies, e also run the “ghunt.py” to check a random e-mail. But one thing I noticed was, every time I run the “check_and_gen.py”, it tries to find e then starts downloading the chromedriver. And there was one time where it downloaded the chromedriver while running the “ghunt.py” command.

Anything else than the above, it is working like a charm.

Thank you!

Best Regards,
